tmt/utils/__init__.py

```diff
@@ -2158,7 +2158,7 @@ class _MultiInvokableCommonMeta(_CommonMeta):
     """
 
     # N805: ruff does not recognize this as a metaclass, `cls` is correct
-    def __init__(cls, *args: Any, **kwargs: Any) -> None:  # noqa: N805
+    def __init__(cls, *args: Any, **kwargs: Any) -> None:
```

Comment:
Comment above probably no longer applies

Reply:
addressed in https://github.com/teemtee/tmt/pull/3142/commits/feeabd484f3a97e109a126f465d56cebddda694c